cmd/buildah/config.go

```diff
@@ -17,99 +17,71 @@ const (
 )
 
 var (
-	configurationFlags = []cli.Flag{
+	configFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "name or ID of the working container",
+		},
+		cli.StringFlag{
+			Name:  "root",
+			Usage: "root directory of the working container",
+		},
+		cli.StringFlag{
+			Name:  "link",
+			Usage: "symlink to the root directory of the working container",
+		},
 		cli.StringFlag{
 			Name:  "author",
-			Usage: "image author contact information",
+			Usage: "image author contact `information`",
 		},
 		cli.StringFlag{
 			Name:  "created-by",
-			Usage: "description of how the image was created",
+			Usage: "`description` of how the image was created",
 			Value: DefaultCreatedBy,
 		},
 		cli.StringFlag{
 			Name:  "arch",
-			Usage: "image target architecture",
+			Usage: "image target `architecture`",
 		},
 		cli.StringFlag{
 			Name:  "os",
-			Usage: "image target operating system",
+			Usage: "image target operating `system`",
 		},
 		cli.StringFlag{
 			Name:  "user",
-			Usage: "user to run containers based on image as",
+			Usage: "`user` to run containers based on image as",
 		},
 		cli.StringSliceFlag{
 			Name:  "port",
-			Usage: "port to expose when running containers based on image",
+			Usage: "`port` to expose when running containers based on image",
 		},
 		cli.StringSliceFlag{
 			Name:  "env",
-			Usage: "environment variable to set when running containers based on image",
+			Usage: "environment `variable` to set when running containers based on image",
 		},
 		cli.StringFlag{
 			Name:  "entrypoint",
-			Usage: "entry point for containers based on image",
+			Usage: "`entry-point` for containers based on image",
```

Comment:
It's an attempt to keep the flag value in the --help output one word instead of two; removing it.